discovery-provider/default_config.ini

```diff
@@ -4,6 +4,7 @@ loglevel_flask = DEBUG
 loglevel_celery = INFO
 block_processing_window = 50
 blacklist_block_processing_window = 600
+peer_refresh_interval = 3000
```

Comment:
@hareeshnagaraj the intervals are so different between .env and here. 10 vs 3000. also is this in sec or ms?

Reply:
seconds, 10 is for dev only and shouldn't be deployed

Reply:
@hareeshnagaraj if this is in seconds isn't this 50 minutes? is that the ttl you wanted?